geotrek/signage/migrations/0013_auto_20200423_1255.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.0.13 on 2020-04-23 12:55
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def delete_force(apps, schema_editor):
+    # We can't import Infrastructure models directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Blade = apps.get_model('signage', 'Blade')
+    for blade in Blade.objects.filter(deleted=True):
+        blade.delete()
```

Comment:
Pas besoin d'une itération, tu peux faire directement `Blade.objects.filter(deleted=True).delete()`